spring-cloud-starter-task-spark-client/README.adoc

```diff
@@ -1,25 +1,22 @@
 //tag::ref-doc[]
 = Spark Client Task
 
-TODO
+This task is intended to launch a Spark application from a task. The task submits the Spark application for local execution. This task is appropriate for a local deployment where any local file references can be resolved. It is not meant for any type of cluster deployments. 
```

Comment:
First sentence should read: `This task is intended to launch a Spark application.` Will fix on merge